identity/client/src/configuration/env.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda
+ * Services GmbH under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file except in compliance with the Camunda License 1.0.
+ */
+
+interface AppWindow {
+  env: Record<string, string>;
+}
+
+const { env } = window as unknown as AppWindow;
```

Comment:
Just to ensure we're all on the same page, I wanted to point out that this approach is slightly different from the implementation in our other web apps. In **Operate** and **Tasklist**, the `window` variable is named `clientConfig`, and environment variables are kept separate from the client configuration.

I see the benefit of the current approach of allowing us to configure the front-end directly from the environment variables, so I do not mind keeping it. We can however think about aligning the name to be `clientConfig` everywhere for example.

FYI @vsgoulart @Ben-Sheppard

Reply:
I talked more in depth with @Ben-Sheppard about this and we're renaming the `window.env` here to `window.clientConfig` to match the setup used in the other webapps.